src/phoenix/datasets/fixtures.py

```diff
@@ -414,8 +374,59 @@ def load_example(use_case: str) -> ExampleDatasets:
 
     """
     fixture = _get_fixture_by_name(use_case)
-    primary_dataset, reference_dataset = download_fixture_if_missing(use_case)
+    primary_dataset, reference_dataset = get_datasets(use_case)
     print(f"📥 Loaded {use_case} example datasets.")
     print("ℹ️ About this use-case:")
     print(fixture.description)
     return ExampleDatasets(primary=primary_dataset, reference=reference_dataset)
+
+
+class Metadata(NamedTuple):
+    path: str
+    mediaLink: str
+    md5Hash: str
+
+    def save_media(self, location: Path) -> Path:
```

Comment:
nit: media makes me think video / audio. Something like data or artifacts might be more appropriate